relationships.go

```diff
@@ -0,0 +1,66 @@
+package dbl
+
+// Relationships help to store the relationships for an Entry
+type Relationships []Relationship
+
+// Append will append a set of relationship IDs for a given relationship
+func (r *Relationships) Append(relationshipIDs ...string) {
```

Comment:
Not sure if you need a pointer to slice here. slices are of reference type.

Reply:
Yep! If you don't reference it as a pointer, then the append function has to return a slice. This allows us to append without needing to return a slice such as:

```go
// Append will append a set of relationship IDs for a given relationship
func (r Relationships) Append(relationshipIDs ...string) 

func (e *Entry) GetRelationships() (r Relationships){
    r = r.Append("foo")
    r = r.Append("bar")
    r = r.Append("baz")
    return
}
```

Although the above example feels more like standard Go slices, it also feels more error prone. This convenience method cleans up the code and makes it more durable.

Reply:
I like the inline approach, less to remember in practice